### DISCUSSION TITLE
Can not write changes back to Data Lakehouse, or to Semantic model inheriting from Lakehouse.

### DISCUSSION THREAD

**Discussion by Erik-Olne** [TabularEditor/TabularEditor3#1195] [🙋♀️TE3 Q&A] [answered]
This may be a noob question, as I am quite new to this. But I have a Lakehouse with many Gen2 flows feeding data into it, and I have a Semantic model created from the Lakehouse interface with tables selected from the Lakehouse.
I have an active Fabric trail, but we do not have a Premium workspace.
I have enabled all the read/wright rights I can find, both for the workspace itself, and for both the Datalakehouse, and the Semantic model.
Connecting to both the Lakehouse, or the Semantic model works fine, but I am unable to save changes back to either connection with this error:
![image](https://github.com/TabularEditor/TabularEditor3/assets/153501979/97373025-6bf9-4e63-93e3-17872fb4e903)

Am I doing anything wrong? or does the Fabric Trial subscription not support this?

Also, I am new to Github. I did try to search for similar tickets, but I'm not sure I I searched correctly. sorry if this is already answered.

**Comment by otykier** [comment, marked as answer]
Hi @Erik-Olne 
Are you attempting to make changes to the _default_ semantic model created for your lakehouse? If so, please be aware that [default semantic models](https://learn.microsoft.com/en-us/fabric/data-warehouse/semantic-models) do not support modifications made from external tools through the XMLA endpoint. As such, if you want to use Tabular Editor to make changes to your Direct Lake model, you'll have to create a new model through the Power BI Service, or [from scratch using Tabular Editor 3](https://blog.tabulareditor.com/2023/09/26/fabric-direct-lake-with-tabular-editor-part-2-creation/).

**Reply by Erik-Olne** [reply]
Thank you for your reply.
Yes, that was indeed what I was trying to do. I beleive I have tried creating a model from PowerBI as well, but I am still new to much of this, so I might done it incorrectly.
I will try creating a model using Tabular Editor according to your link, and report back.

**Reply by Erik-Olne** [reply]
That did the trick! you are a lifesaver! Thank you :-)
